examples/dihedral/state.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Do not include state file, but put an equilibrated start position in `dihedral.yml` imput file.

Reply:
Fixed with 8ff5d87.